azurerm/resource_arm_database_vulnerability_assessment_rule_baseline.go

```diff
@@ -0,0 +1,238 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/sql/mgmt/2017-03-01-preview/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmDatabaseVulnerabilityAssessmentRuleBaseline() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineCreateUpdate,
+		Read:   resourceArmDatabaseVulnerabilityAssessmentRuleBaselineRead,
+		Update: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineCreateUpdate,
+		Delete: resourceArmDatabaseVulnerabilityAssessmentRuleBaselineDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"server_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: azure.ValidateMsSqlServerName,
+			},
+
+			"database_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: azure.ValidateMsSqlDatabaseName,
+			},
+
+			"rule_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"baseline_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  string(sql.VulnerabilityAssessmentPolicyBaselineNameDefault),
+				ValidateFunc: validation.StringInSlice([]string{
+					string(sql.VulnerabilityAssessmentPolicyBaselineNameDefault),
+					string(sql.VulnerabilityAssessmentPolicyBaselineNameMaster),
+				}, true),
```

Comment:
Could we make this case sensitive:
```suggestion
				}, false),
```